app/models/quarter.rb

```diff
@@ -0,0 +1,39 @@
+class Quarter
+  def self.create(date:)
+    new(date: date)
+  end
```

Comment:
Do we need this `create` method? Could we just use `new`?

Reply:
Yup removed that unnecessary create method.